api/server/router/container/container.go

```diff
@@ -60,6 +60,7 @@ func (r *containerRouter) initRoutes() {
 		// PUT
 		local.NewPutRoute("/containers/{name:.*}/archive", r.putContainersArchive),
 		// DELETE
+		local.NewDeleteRoute("/containers/_all", r.deleteAllContainers),
```

Comment:
I'm not sure either if we should add `_all` to the endpoint. I understand the concerns of inadvertently removing all containers if the client makes a "whoopsie" and doesn't include a name/id. However I'm not sure this should be a concern of the API, it feels like something the client *consuming* the should handle.

I'm not fully convinced on making a difference between trailing slash/no trailing slash (sorry Tibor :smile:).

*If* we *really* want something to protect clients from accidentally performing a batch-delete, I'd be more in favor of a query-parameter (e.g. `?all=1`), but I'd be fine with no protection at all and keeping the responsibility to the client